Makefile

```diff
@@ -36,92 +46,44 @@ install: telegraf
 test:
 	go test -short ./...
 
+fmt:
+	go fmt -w ./...
+
+fmtcheck:
+	@echo '[INFO] running go fmt'
+	@gofmt -l $(GOFILES);
```

Comment:
This will list the files, but it still returns zero, we will need it to fail the build.